src/IJulia.jl

```diff
@@ -289,7 +290,8 @@ push_preexecute_hook(f::Function) = push!(preexecute_hooks, f)
 Remove a function `f()` from the list of functions to
 execute before executing any notebook cell.
 """
-pop_preexecute_hook(f::Function) = splice!(preexecute_hooks, findfirst(equalto(f), preexecute_hooks))
+pop_preexecute_hook(f::Function) =
+    splice!(preexecute_hooks, findlast(isequal(f), preexecute_hooks))
```

Comment:
`Compat.`?

Reply:
Doesn’t need it. If f isn’t found, whether findlast returns 0 or nothing splice! will throw.

Reply:
(But I guess it wouldn't hurt to make sure it throws the same exception on 0.6 and 0.7.)